assets/css/style.css

```diff
@@ -0,0 +1,136 @@
+/* font styling */
+h1,
+h2,
+h3,
+h4,
+p,
+ul,
+li {
+font-family: 'Open Sans', sans-serif;
```

Comment:
BTW, as you make changes based on these comments, you can continue pushing to the TestArea branch. This PR will update with the changes accordingly.